torch/_dynamo/eval_frame.py

```diff
@@ -786,6 +786,8 @@ def export(
     if pre_autograd:
         assert aten_graph, "pre_autograd=True can only be used when aten_graph=True"
     f = innermost_fn(f)
+    call_to_inspect = f.forward if isinstance(f, torch.nn.Module) else f
```

Comment:
Is `self` eliminated from the signature in this case? (That's good, but not sure how that happens.)

Reply:
yeah, "self" is ignored by inspect.signature automatically. See here https://github.com/python/cpython/blob/3.11/Lib/inspect.py#LL2449C1-L2452C46